Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -192,42 +215,44 @@ def ShowMof(op, ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
 def Set(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
     ShowMof('SET', ConfigType, ConfigID, Contents, Ensure, ContentChecksum)
     retval = 0
-    if ConfigType != 'UpdatedAgentConfig':
```

Comment:
But what is the need of moving it down? If the contract is not adhered, we should immediately return without further processing.